Detections/MultipleDataSources/SuspiciousLoginfromDeletedExternalIdentities.yaml

```diff
@@ -0,0 +1,58 @@
+id: defe4855-0d33-4362-9557-009237623976
+name: Suspicious modification of  Global Administrator user properties
+description: |
+  ' This query will detect logins from guest account which was recently deleted. 
+  For any successful logins from deleted identities should be investigated further if any existing user accounts have been altered or linked to such identity prior deletion'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: AzureActiveDirectory
+    dataTypes:
+      - AuditLogs
+  - connectorId: AzureActiveDirectory
+    dataTypes:
+      - SigninLogs
```

Comment:
Wondering if we need SigninLogs here. We should probably also put in the connector details for IdentityInfo table.